react/BottomDrawer/styles.styl

```diff
@@ -1,9 +1,10 @@
 @require 'tools/mixins'
 @require 'settings/spaces'
 @require 'settings/z-index'
+@require 'generic/animations'
 
 .with-transition
-    transition transform .1s ease-out
+    @extends $with-transition
```

Comment:
The placeholder name could be a bit more descriptive, it is impossible when reading BottomDrawer to infer what's going on.

```
@extends $transition-transform-ease-out
```

?

Reply:
yep thanks!